controllers/pvc.go

```diff
@@ -205,40 +201,3 @@ func (r *DataMoverBackupReconciler) getSourcePVC() (*corev1.PersistentVolumeClai
 
 	return pvc, nil
 }
-
```

Comment:
Since we remove wait, should [this](https://github.com/konveyor/volume-snapshot-mover/pull/20/files#diff-f62131094710ef63a76f5e484071a4e7254ff8a2d176f0957e2dd677495eff04R28-R29) be changed to return `false,nil`? Also, the log should be Info IMO

Reply:
Line no: 28-29 in pvc.go. The link is not redirecting correctly.